app/src/main/java/uk/gov/onelogin/login/ui/welcome/WelcomeScreenViewModel.kt

```diff
@@ -53,11 +53,8 @@ class WelcomeScreenViewModel @Inject constructor(
             context.resources.getString(R.string.openIdConnectClientId)
         }
 
-        val scopes = if (featureFlags[StsFeatureFlag.STS_ENDPOINT]) {
-            listOf(LoginSessionConfiguration.Scope.OPENID)
-        } else {
-            listOf(LoginSessionConfiguration.Scope.STS)
-        }
+        val scopes = listOf(LoginSessionConfiguration.Scope.OPENID)
```

Comment:
So we don't use STS anymore?

Reply:
this seems to be the only way to get the call to auth to work